### DISCUSSION TITLE
form create and update

### DISCUSSION THREAD

**Discussion by rcarvalho2clicks** [Budibase/budibase#11339] [Help] [answered]
I would like to know if it is possible to create and update in the same form?

**Comment by tglynx** [comment, marked as answer]
Sure! You can switch the type of a form using conditions! 
See screenshot for an example: 
![Bildschirm­foto 2023-07-24 um 21 22 22](https://github.com/Budibase/budibase/assets/30650435/70c39f29-7388-4ae3-ae6d-4dba0e836125)

1. Select the Form component
2. click on 'conditions' 
3. create appropriate condition to switch the form type from create to update based on your use case

**Comment by rcarvalho2clicks** [comment]
thanks for the help

Em seg., 24 de jul. de 2023 às 16:24, Michael Sommer <
***@***.***> escreveu:

> Sure! You can switch the type of a form using conditions!
> See Screenshot for an example:
> [image: Bildschirm­foto 2023-07-24 um 21 22 22]
> <https://user-images.githubusercontent.com/30650435/255692399-70c39f29-7388-4ae3-ae6d-4dba0e836125.png>
>
>    1. Select the Form component
>    2. click on 'conditions'
>    3. create appropriate condition to switch the form type from create to
>    update based on your use case
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/11339#discussioncomment-6532993>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAG55XU3J2AUQNC4WNM5B73XR3DVZANCNFSM6AAAAAA2V5TIC4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
<http://www.jpiaget.com.br> *Rodrigo Damasceno de Carvalho*
Supervisor de Tecnologia da Informação

(11) 4367-8303 / (11) 99124-9874
***@***.***
